pkg/tkctl/cmd/list/list.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2019. PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package list
+
+import (
+	"github.com/golang/glog"
+	"github.com/pingcap/tidb-operator/pkg/apis/pingcap.com/v1alpha1"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/config"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/readable"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	kubeprinters "k8s.io/kubernetes/pkg/printers"
+)
+
+const (
+	listLongDesc = `
+		List all tidb clusters.
+
+		Prints a table of the general information about each tidb cluster. By specifying
+		namespace or label-selectors, you can filter clusters.
+`
+	listExample = `
+		# list all clusters and sync to local config
+		tkc list
+
+		# filter by namespace
+		tkc list --namespace=foo
+
+		# get tidb cluster in all namespaces
+		tkc list -A
```

Comment:
tkc list --all-namespace ?

Reply:
`--all-namespaces` is the longer version of `-A`